src/components/pages/RegisterPage.vue

```diff
@@ -0,0 +1,316 @@
+                            <input 
+                                type="text" 
+                                name="email"
+                                placeholder="Confirm Email address" 
+                                class="form-input" 
+                                v-model="emailVerify"
+                                @input="validateFormEM"
+                                required
+                            />
+                        </div>
+
+                        <!-- Valid Password -->
+                        <span class="invalid-feedback" v-if="validation.invalid.password">
+                            {{ validation.invalid.password }}
+                        </span>
+                        <!-- Invalid password -->
+                        <span class="valid-feedback" v-if="validation.valid.password">
+                            {{ validation.valid.password }}
+                        </span>
+                        <div class="form-field">
+                            <label class="lock" for="login-password"><span class="hidden">Password</span></label>
+                            <input 
+                                id="password" 
+                                type="password"
+                                name="password"
+                                class="form-input" 
+                                placeholder="Password" 
+                                v-bind:class="{
+                                    'is-valid' : validation.valid.password,
+                                    'is-invalid' : validation.invalid.password
+                                }"
+                                v-on:focus="validation('password')"
+                                v-model="password"
+                                required
+                            />
+                        </div>
+
+                        <!--- Invalid password confirmation -->
+                        <span class="invalid-feedback" v-if="errorMessagePW">
+                            {{ errorMessagePW }}
+                        </span>
+                        <div class="form-field">
+                            <label class="lock-confirm" for="login-password"><span class="hidden">Confirm Password</span></label>
+                            <input 
+                                id="login-password" 
+                                type="password" 
+                                class="form-input" 
+                                placeholder="Confirm Password" 
+                                v-model="passwordVerify"
+                                @input="validateFormPW"
+                                required
+                            />
+                        </div>
+                            
+                        <div class="form-field">
+                            <input type="submit" value="Register Now">
```

Comment:
You need to add a reference to the method that will be called when the user clicks Submit.